### PR TITLE
add ctrl + shift + A shortcut for desktop and tooltip to select all checkbox

### PR DESCRIPTION
**Pull request by ganthern**

### CODE REVIEW

**Comment by charlag** [comment]
People do use Ctrl-A to select all text in the mail, it would conflict, we should look for another one

**Comment by ganthern** [comment]
that does not work for firefox at least, but we can still use something else

**Review by charlag — CHANGES_REQUESTED** [review]

File: src/gui/base/List.ts [hunk 1]
```diff
@@ -1618,12 +1626,22 @@ export function listSelectionKeyboardShortcuts<T extends ListElement, R extends
...
 			help: "addNext_action",
+			enabled: mapLazily(list, (list) => list.config.multiSelectionAllowed),
+		},
+		{
+			key: Keys.A,
+			ctrl: true,
+			shift: true,
+			exec: mapLazily(list, (list) => (list.isAllSelected() ? list.selectNone() : list.selectAll())),
+			help: "selectAllLoaded_action",
+			enabled: mapLazily(list, (list) => isDesktop() && list.config.multiSelectionAllowed),
```
[the remark this answers is not in the document]
Comment (ganthern) [reply src/gui/base/List.ts:1644]: it also shadows the search in tabs shortcut in chrome and in safari

[the remark this answers is not in the document]
Comment (charlag) [reply src/gui/base/List.ts:1644]: then let's come up with something bespoke and random, like "O" for calendar

**Review by charlag — APPROVED** [review]

**Pull request merged**